build/ferretdb/eval-dev.Dockerfile

```diff
@@ -129,10 +129,6 @@ EXPOSE 27017 27018 8088
 ENV GOCOVERDIR=/tmp/cover
 ENV GORACE=halt_on_error=1,history_size=2
 
-# FIXME
-# do not allow "trust" authentication for local connections while initializing
-# ENV POSTGRES_INITDB_ARGS="--auth=scram-sha-256"
-
 ENV FERRETDB_STATE_DIR=/tmp/state
```

Comment:
Let's also use `/state` as in other Dockerfiles and our docs (making sure that it actually works, of course)

Reply:
Yes of course.